tests/spread/plugins/craft-parts/kernel/snapcraft.yaml

```diff
@@ -0,0 +1,77 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2021 Canonical Ltd
+name: pc-kernel
+summary: Generic Ubuntu pc-kernel
+description: The generic Ubuntu pc-kernel
```

Comment:
Is this still an accurate statement?  I remember you saying you trimmed many options in the kernel config.

I know the snapcraft.yaml's in spread tests are often used as references, so I don't want to mislead anyone into thinking this is _the_ generic Ubuntu pc-kernel.